cdflow_commands/ecs_monitor.py

```diff
@@ -153,6 +196,13 @@ class TimeoutError(UserError):
     )
 
 
+class FailedTasksError(UserError):
+    _message = (
+        ('Deployment failed - tasks (containers) are '
+         'failing to pass healthcheck')
```

Comment:
I think this could be a little presumptuous - the health check is certainly one thing that can cause containers to be stopped but surely not the only thing? We know from our check that the running count decreased so I'd say that's a more reliable thing to state in the error.